### PR TITLE
Add project page design

### PR DESCRIPTION
**Pull request by AlexPoliteRudenko**

### CODE REVIEW

File: src/main/webapp/resources/ProjectPage.html [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+
+</div>
+
+<!--Modal-->
+<div class="modal fade" id="modal-invest">
+    <div class="modal-dialog">
+        <div class="modal-content ">
+            <div class="modal-header">
+                <button class="close" type="button" data-dismiss="modal">
+                    <i class="fa fa-close"></i>
```
Comment (Sheptytskyid) [inline src/main/webapp/resources/ProjectPage.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

File: src/main/webapp/resources/ProjectPage.html [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle" data-toggle="collapse"
+                    data-target="#responsive-menu">
+                <span class="sr-only">Menu</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="#">Great<i class="fa fa-rocket fa-lg" aria-hidden="true">
```
Comment (Sheptytskyid) [inline src/main/webapp/resources/ProjectPage.html]: MINOR Replace this <i> tag by <em>. [rule](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by zozich — APPROVED** [review]
👍

**Pull request merged**